picasso/chain.json

```diff
@@ -32,7 +32,7 @@
     ]
   },
   "codebase": {
-    "git_repo": "https://github.com/notional-labs/composable-centauri",
+    "git_repo": "https://github.com/composable/composable-cosmos",
     "recommended_version": "v6.5.3",
```

Comment:
v6.6.1 please

Reply:
Not sure if this will be the same release for mainnet, will add once we confirm it